keps/sig-node/1769-memory-manager/README.md

```diff
@@ -543,17 +544,25 @@ _This section must be completed when targeting beta graduation to a release._
 * **How can a rollout fail? Can it impact already running workloads?**
   Try to be as paranoid as possible - e.g., what if some components will restart
    mid-rollout?
+  It is possible that the state file will have inconsistent data during the rollout, because of the kubelet restart, but
+  you can easily to fix it by removing memory manager state file and kubelet restart. It should not affect any running 
+  workloads.
+  
 
 * **What specific metrics should inform a rollback?**
+None.
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
   Describe manual testing that was done and the outcomes.
   Longer term, we may want to require automated upgrade/rollback tests, but we
   are missing a bunch of machinery and tooling and can't do that now.
+  Tested it manually by replacing the kubelet binary on the node with the `Static` memory manager policy, but I failed
+  to find correct procedure how to test upgrade from 1.21 to my custom build with updated kubelet binary.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
 fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+  Even if applying deprecation policies, they may still surprise some users. 
+  No.
 
 ### Monitoring Requirements
```

Comment:
this section (and I think a couple others) need to be completed.

Reply:
@deads2k It does not deprecate or remove and fields so I added `No`, or should I make it more clear?

Reply:
You appear to be missing monitoring requirements, usage determination, SLOs, dependencies, etcd